backend/business/core/collectors/github/fetchers/org_info.go

```diff
@@ -38,31 +50,31 @@ type orgInfoQuery struct {
 	Organization orgInfo `graphql:"organization(login:$organizationLogin)"`
 }
 
-func NewOrgInfo(client *githubv4.Client, orgName string) *OrgInfo {
-	return &OrgInfo{client: client, orgName: orgName}
+func NewOrgInfo(client *githubv4.Client, orgName string) *OrgInfoFetcher {
+	return &OrgInfoFetcher{client: client, orgName: orgName}
 }
 
-func (o *OrgInfo) Fetch(ctx context.Context) (string, error) {
+func (oif *OrgInfoFetcher) Fetch(ctx context.Context) (*OrgInfoResult, error) {
 	variables := map[string]interface{}{
-		"organizationLogin": githubv4.String(o.orgName),
+		"organizationLogin": githubv4.String(oif.orgName),
 	}
 
 	var q orgInfoQuery
 
-	err := o.client.Query(ctx, &q, variables)
+	err := oif.client.Query(ctx, &q, variables)
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 
-	csvString, err := o.formatCSV(q.Organization)
+	result := oif.buildResult(q.Organization)
 
 	if err != nil {
-		return "", err
+		return result, err
 	}
-	return csvString, nil
+	return result, nil
 }
 
-func (f *OrgInfo) formatCSV(data orgInfo) (string, error) {
+func (f *OrgInfoFetcher) formatCSV(data orgInfo) (string, error) {
```

Comment:
Can we remove the various `formatCSV` funcs since we are using JSON now?

Reply:
Yeah, we left these initially in case we want to add CSV support but I think this is a good example of [YAGNI](https://martinfowler.com/bliki/Yagni.html) - will remove!

Reply:
Updated in 1bd8bc5eb947c402a5c2fe080e73008a487b0a53